cardinal/world.go

```diff
@@ -207,6 +208,41 @@ func NewMockWorld(opts ...WorldOption) (*World, error) {
 	return world, nil
 }
 
+func RegisterMessage[In any, Out any](world *World, name string, opts ...message.MessageOption[In, Out]) error {
+	msgType := message.NewMessageType[In, Out](name, opts...)
+	var msg types.Message = msgType
+	err := world.registerMessagesByName(msg)
+	if err != nil {
+		return eris.Wrap(err, "failed to register message")
+	}
```

Comment:
no need for the extra `msg` variable here, the `msgType` _is_ `types.Message` already, so it can be passed it to methods that accept `types.Message`